### ISSUE TITLE
[Visual Requirement - Makecode-Documentation] Luminosity ratio is less than 4.5:1 for for the text 'Flashing heart' displayed on top left Bar

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2797]
**User Experience**
If the luminosity ratio is less than 4.5:1 for all the text content then it will be difficult for the visually impaired user to see what is present on the screen.

**Test  Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool: Color contrast Analyzer

**Repro Steps**
1-Navigate to url:-https://makecode.microbit.org/reference/basic/show-number
2-Launch Color Contrast Analyzer tool using WAT tool bar.
3-Verify, the luminosity ratio is greater than 4.5:1 for Flashing heart' displayed on top left bar or not

**Actual Result**
Luminosity contrast ratio is less than 4.5:1 for the text 'Flashing heart' displayed on top left Bar

**Expected Result**
Luminosity contrast ratio should be greater than 4.5:1 for the text 'Flashing heart' displayed on top left Bar

**Note:**
Same issue appears all the text appears on left bar

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Refer Attachment:**
[MAS 17_color contrastratiofailed.zip](https://github.com/Microsoft/pxt/files/1248531/MAS.17_color.contrastratiofailed.zip)

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/reference/basic/show-number and issue is fixed now. Hence closing the bug
